packages/clay-tabs/src/Item.tsx

```diff
@@ -40,9 +40,9 @@ interface IProps extends Omit<React.HTMLAttributes<HTMLLIElement>, 'onClick'> {
 	>;
 
 	/**
-	 * Registers the `tabKey` for the Tab Item.
+	 * Registers the `tabkey` for the Tab Item.
 	 */
-	tabKey: number;
+	tabkey: number;
```

Comment:
Why the change from `tabKey` to `tabKey`? It is inconsistent with other props like `tabName` and `activeTabKey`?

Reply:
I was getting errors with this property using camel case. See the following stacktrace:
```log

Reply:
I see, because you are including it with `otherProps` and passing those to the `div` element rendered by `TabPane`.

Reply:
I don't think that would be good to remove `otherProps`. I can just extract the value of `tabKey` and don't use. What do you think about it?

Reply:
👍 Yes, that's exactly what I meant by "you should remove it from `otherProps`".